### DISCUSSION TITLE
Fragen zu Menü: "Wechselrichter bearbeiten"

### DISCUSSION THREAD

**Discussion by Fokker-1** [tbnobody/OpenDTU#1170] [Q&A]
Hallo,

Ich habe noch mal eine Frage.
Im Menü "Wechselrichter bearbeiten" gibt es u. a. die beiden folgenden Bereiche:

a) Max. Leistung String x
b) Ertragsversatz String x

Die beiden Eingabefelder sind direkt nebeneinander angeordnet.
- Sind deren Daten daher miteinander korreliert?
- Müssen immer beide Felder ausgefüllt werden?

**zu a) Max. Leistung String x:**
In der Fußnote (*) bei Wp* steht dazu:
"Geben Sie die Wp des Ports ein, um die Einstrahlung zu errechnen."
Im Info-Fenster steht: "... Eingabe ... maximale Leistung der Solarmodule".

Hier soll also - gem. Info-Fenster - die max. Leistung Wp der angeschlossenen **_Solarmodule_** an diesem String eingegeben werden, bei mir also 1 Modul mit 430 Wp.
Mit "Wp des **_Ports_**" ist also nicht die max. zulässige Leistung des **_WR-Ports_** gemeint?

Nun ist mir weiter unklar:
- Welche "Einstrahlung" wird daraus berechnet?
- Wofür werden diese Daten benötigt?

**zu b) Ertragsversatz String x (Offset):**
Damit kann - gemäß Info-Fenster - der "... der Gesamtertragswert des WR auf Null gesetzt werden ...".

Als Beispiel jetzt mal bei mir:
    WR mit 2 Ports, beide belegt.

Hier ist mir unklar, was ich tatsächlich dazu machen muss:
- den **_aktuellen_** Gesamtertragswert des String 1 auslesen und hier eintragen?
- das gleiche bei String 2 machen?

- was ist dann das Ergebnis:
       - Wird damit - gemäß Info-Fenster - der **_Gesamtertragswert des WR_** auf Null gesetzt.
       - Gleichzeitig auch die Gesamtertragswerte auf Null von String 1 und 2?
       - Oder bleiben die Gesamtertragswerte von String 1 und 2 unverändert?

.... aller Anfang ist schwer.

Vielen Dank im Voraus für hilfreiche Informationen!
Entspannten Tag noch!

**Comment by tbnobody** [comment]
> a) Max. Leistung String x

Die aktuelle Leistung des entsprechenden Eingangs geteilt durch die hier eingegebene Maximalleistung entspricht der Einstrahlung:

![image](https://github.com/tbnobody/OpenDTU/assets/1753222/30047be9-df00-462d-b5c5-71f4d75b9f4d)

> b) Ertragsversatz String x

Wenn du deinen Inverter gebraucht gekauft hast und dieser z.B. bereits 200kWh auf dem Zähler hat kannst du hier mit einer Eingabe von -200 den Zähler für dich auf 0 setzen. Den integrierten Zähler des Inverters kann man nicht resetten. Dieser kann aber mit diesem Eingabefeld "virtuell" geändert werden.

**Comment by Fokker-1** [comment]
Guten Morgen,
vielen Dank für die wieder schnelle und ausführliche Antwort!

**zu a) Max. Leistung String x**
Habe ich getestet, super, passt.

**zu b) Ertragsversatz String x**
Auf die Idee, dass man da ein Minuszeichen vor die Zahl setzen muss, wäre ich nicht gekommen.
Vielleicht könnte man diese Angabe ja noch in das Info-Fenster mit aufnehmen.
Habe es so probiert => prima, passt so.

Ein Problem ist dabei allerdings aufgetreten bei meinem Samsung S20FE.
Tippt man in das Eingabefeld, klappt die virtuelle Tastatur am Display zur Eingabe auf. Diese Tastatur enthält aber keine Möglichkeit, ein Minuszeichen einzugeben - zumindest habe ich keine gefunden. Aktiv sind nur die Ziffern und der Punkt.

![IMG-20230726-WA0002](https://github.com/tbnobody/OpenDTU/assets/140426500/e836ddab-d18e-483b-9bf4-a415c54358d3)

Ich habe es dann mit dem Laptop getestet, da geht es natürlich.

Noch mal danke und viele Grüße!

**Comment by Alf-Ele** [comment]
Danke, das hat mir auch geholfen. Das Minus Zeichen fehlte bei mir auch auf der Samsung Tastatur, musste es mit Copy/Paste einfügen

**Comment by Fokker-1** [comment]
Danke für die Info zur Samsung-Tastatur.
Damit sollte klar sein, dass es sich nicht um ein spezielles Problem bei meinem Handy handelt.

Frage mal dazu:
Besteht evtl. eine Möglichkeit, dem Eingabefenster für den Ertrags-Versatz ein bestimmtes Merkmal zuzuordnen, so dass sich im Handy eine andere virtuelle Tastatur öffnet, die nicht nur Ziffern und den Punkt beinhaltet? Diese ist ja vorhanden, vielleicht muss sie "nur" konkret aufgerufen werden.

Schon mal vielen Dank im Voraus und viele Grüße!

**Comment by StupidHumansInc** [comment]
Hi,
erst mal Danke für die super Arbeit!

Meine Frage: Gibt man bei **Max. Leistung String**  die Werte der STC oder die der NMOT ein?

Vielen Dank und Grüße!

**Reply by Alf-Ele** [reply]
Ich würde STC nehmen, 
zumindest passt dies bei mir ganz gut.
Mit dem Wert wird danach ja die Einstrahlung berechnet in %, bei NMOT würden bei mir Werte weit über 100% herauskommen.
Ist aber nur eine Anzeigethematik.
